iXBRLViewerPlugin/viewer/src/js/unit.js

```diff
@@ -48,19 +60,6 @@ export class Unit {
         return this._measure;
     }
```

Comment:
I was wondering if this could be removed, but it's still used by `Fact.getScaleLabel()`.  Looking at it, it seems that it only gets used if the fact is monetary, so I think if we update `isMonetary()` to require `isSimple()` as per other comments, then this is fine.